app/components/wallet/transactions/Transaction.js

```diff
@@ -160,19 +164,19 @@ export default class Transaction extends Component {
           <div className={styles.togglerContent}>
             <div className={styles.header}>
               <div className={styles.title}>
-                {data.type === 'expend' ?
-                  intl.formatMessage(messages.adaSent) :
-                  intl.formatMessage(messages.adaReceived)
+                {data.type === transactionTypes.EXPEND ?
+                  intl.formatMessage(messages.sent, { currency }) :
+                  intl.formatMessage(messages.received, { currency })
                 }
               </div>
-              <div className={styles.amount}>{data.amount.toFormat(DECIMAL_PLACES_IN_ADA)}
-                <SvgInline svg={adaSymbol} className={styles.currencySymbol} />
+              <div className={styles.amount}>{formattedWalletAmount(data.amount)}
```

Comment:
@DominikGuzei please use `formattedAmountWithoutTrailingZeros` instead of `formattedWalletAmount` here as we need to show exact transaction amount - otherwise we will end up with showing `< 0.000001` for very small ones :(

![screen shot 2017-11-08 at 09 35 41](https://user-images.githubusercontent.com/376611/32539445-b876f80c-c469-11e7-9a2d-983020f41d77.png)